src/cli/cluster/cluster_manager.js

```diff
@@ -86,7 +86,6 @@ module.exports = class ClusterManager {
     const fromRoot = require('../../utils/fromRoot');
 
     this.watcher = chokidar.watch([
-      'src/plugins',
```

Comment:
@spalger Yes I thought about using uniq. But the default options use fromRoot which are absolute paths and these are relative, we should change all paths to relative or absolute. What do you think?

We can upgrade chokidar also !